job-server-extras/src/main/scala/spark/jobserver/python/PythonJob.scala

```diff
@@ -57,7 +57,8 @@ case class PythonJob[X <: PythonContextLike](eggPath: String,
     logger.info(s"Running $modulePath from $eggPath")
     val ep = endpoint(sc, runtime.contextConfig, runtime.jobId, data)
     val server = new GatewayServer(ep, 0)
-    val pythonPath = (eggPath +: sc.pythonPath).mkString(":")
+    val pythonPathDelimiter : String = if(System.getProperty("os.name").indexOf("Win") >= 0) ";" else ":"
```

Comment:
Thanks, @velvia! 
Should I update the PR with the styling fixed? Also forgot to add a note to readme.md (Contribution and Development section)